src/chrome/content/rules/EOL.org.xml

```diff
@@ -0,0 +1,22 @@
+<!--
+	Invalid certificate:
+		blog.eol.org
+	
+	Refused:
```

Comment:
`cephbase.eol.org`

Reply:
This hosts times out on both `http` and `https` for me. Is it working for you?

Reply:
I can reproduce the timeout, but this could also be a temporary issue. I think it worked on http and was refused on https when I tested it.